app-ios/tutanota/Sources/Files/IosFileFacade.swift

```diff
@@ -37,7 +37,23 @@ class IosFileFacade : FileFacade {
   
   func openFileChooser(_ boundingRect: IpcClientRect, _ filter: [String]?) async throws -> [String] {
     let anchor = CGRect(x: boundingRect.x, y: boundingRect.y, width: boundingRect.width, height: boundingRect.height)
-    return try await self.chooser.open(withAnchorRect: anchor)
+    let files = try await self.chooser.open(withAnchorRect: anchor)
+    var returnfiles = [String]()
+    for file in files {
+      let fileUrl = URL(fileURLWithPath: file)
+      let isDirectory: Bool = (try? fileUrl.resourceValues(forKeys: [.isDirectoryKey]))?.isDirectory == true
+      // This should only be for files, but sometimes a directory masquerading as a file can slip through (such as a .band file).
+      // In those cases we just zip and add it.
+      if isDirectory {
+        guard let destinationPath = try? zipFile(fileUrl: fileUrl) else {
```

Comment:
I don't think we should quietly swallow these errors, we should rethrow it (with just `try`)